Helper/Config.php

```diff
@@ -18,6 +18,7 @@
 namespace Bolt\Boltpay\Helper;
 
 use Bolt\Boltpay\Model\Api\Data\BoltConfigSettingFactory;
+use Codeception\Module\OrderHelper;
```

Comment:
@johnmccombs1 Is this intentional?  I believe it can unintentionally cause errors in some cases.

Reply:
@mccottry it's just my typo. Updated